src/test/scala/chiseltest/tests/FaultLocatorTest.scala

```diff
@@ -41,13 +54,12 @@ class FaultLocatorTest extends AnyFlatSpec with ChiselScalatestTester with Match
         c.out.expectDequeueNow(true.B)
       }
     }
-    // Only check the filename to avoid this being too brittle as implementation changes
-    exc.failedCodeFileNameAndLineNumberString.get should startWith ("DecoupledDriver.scala:")
-    exc.getMessage should include regex ("""\(lines in FaultLocatorTest\.scala:[^\)]*41.*\)""")
+      // Only check the filename to avoid this being too brittle as implementation changes
+      .expects.head.trace.toString should include regex ".*(DecoupledDriver.scala:\\d+)"
```

Comment:
Style: I don't like the comment inbetween the object and the call. Similar below.

I'm not sure if we just want to check the name, since we want to make sure it's locating the right part of that file.

But if you only wanted to check the name, you could make that clear in the test name / description, instead of a comment.